src/client.ts

```diff
@@ -19,12 +24,18 @@ interface ClientConfigOptions {
   clientWindow?: Window;
 }
 
+/**
+ * A strictly-typed event handler for publication messages.
+ */
+type ClientEventEmitter = EnvDataEventEmitter & PublicationEventEmitter;
```

Comment:
Most of this will change with the changes to PubSub, but I wanted to get the init api flowing through events.